### ISSUE TITLE
vagrant up fails to bring Windows up when WinRM is not ready

### ISSUE THREAD

**Issue by netoarmando** [freeipa/freeipa-pr-ci#307]
Eventually PR-CI fails to bring Windows box up, returning `The device is not ready`:
```java
2019-08-29 09:50:20,148    DEBUG  An error occurred while executing the action on the 'ad-root'
2019-08-29 09:50:20,148    DEBUG  machine. Please handle this error then try again:
2019-08-29 09:50:20,148    DEBUG  
2019-08-29 09:50:20,148    DEBUG  An error occurred executing a remote WinRM command.
2019-08-29 09:50:20,149    DEBUG  
2019-08-29 09:50:20,149    DEBUG  Shell: Cmd
2019-08-29 09:50:20,149    DEBUG  Command: hostname
2019-08-29 09:50:20,150    DEBUG  Message: [WSMAN ERROR CODE: 2147942421]: <f:WSManFault Code='2147942421' Machine='192.168.121.81' xmlns:f='http://schemas.microsoft.com/wbem/wsman/1/wsmanfault'><f:Message><f:ProviderFault path='%systemroot%\system32\winrscmd.dll' provider='Shell cmd plugin'>The device is not ready. </f:ProviderFault></f:Message></f:WSManFault>
```
It happened in the past, but I haven't seen it for a long time, it seems to be related to https://github.com/test-kitchen/test-kitchen/issues/891.

Back then I tried to generate my own windows image to fix this WinRM issue by adding Windows registry entries to only enable this service when Windows is fully ready.

Another way to solve this is by running `vagrant up` again after a few seconds.
This [patch](https://github.com/freeipa/freeipa-pr-ci/files/3592388/vagrantup.patch.txt) tries to fix that.

However, this issue stopped to occur in our production runners, so we cannot reproduce this any more. 

~Marking this as `won't fix` for now.~

**Comment by netoarmando** [comment]
This issue is still affecting one runner (at least).

**Comment by netoarmando** [comment]
For reference, after testing a quick and dirty solution, which consisting in running `vagrant up` once again after a few seconds, I get an unrelated and already fixed issue: http://freeipa-org-pr-ci.s3-website.eu-central-1.amazonaws.com/jobs/c9616e0a-d8b1-11e9-a5ca-fa163edf5348/.

**Comment by netoarmando** [comment]
It seems this was fixed by #310. Closing this for now.
